backend/alias/alias.go

```diff
@@ -0,0 +1,43 @@
+package alias
+
+import (
+	"errors"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/ncw/rclone/fs"
+	"github.com/ncw/rclone/fs/config"
+)
+
+// Register with Fs
+func init() {
+	fsi := &fs.RegInfo{
+		Name:        "alias",
+		Description: "Alias for a existing remote",
+		NewFs:       NewFs,
+		Options: []fs.Option{{
+			Name: "remote",
+			Help: "Remote to alias.\nNormally should contain a ':' and a path, eg \"myremote:path/to/dir\",\n\"myremote:bucket\" or maybe \"myremote:\".",
```

Comment:
Looking at the code I think it should allow just putting a path in, so `/path/to/folder` could be an option here too.

Reply:
It's already possible to enter a local path. 
The option is mainly copied from the crypt backend. 
It is also possible specify a local path there (i'm pretty sure).

I will change the documentation with the next commit.

Reply:
I probably didn't put that docs in the crypt as I didn't want to confuse people, whereas I think making local aliases will be an important use of this :-)